library/ssl_tls13_server.c

```diff
@@ -585,6 +585,11 @@ static int ssl_tls13_parse_client_hello( mbedtls_ssl_context *ssl,
      */
     ssl->tls_version = MBEDTLS_SSL_VERSION_TLS1_3;
 
+#if defined(MBEDTLS_SSL_SESSION_TICKETS)
+    /* Store minor version for later use with ticket serialization. */
+    ssl->session_negotiate->tls_version = MBEDTLS_SSL_VERSION_TLS1_3;
+#endif /* MBEDTLS_SSL_SESSION_TICKETS */
```

Comment:
```suggestion
#endif
```

(the `MBEDTLS_SSL_SESSION_TICKETS` comment not needed as so close to the `#if`)